parlai/tasks/coqa/build.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+import json
+
+
+def make_parlai_format(outpath, dtype, data):
```

Comment:
i wonder if we should save the parlai format and directly download it from AWS instead of having the users rebuild every time. thoughts @stephenroller? this is what i have traditionally done in the past.

Reply:
It depends on the copyright mostly. In this case I was thinking we might need the original files to squeeze into wizard format or similar.